front/tests/003-study-management.spec.ts

```diff
@@ -7,24 +7,27 @@ import study from './assets/operationStudies/study.json';
 import { StudyPage } from './pages/study-page-model';
 import { ProjectPage } from './pages/project-page-model';
 import PlaywrightCommonPage from './pages/common-page-model';
-import { deleteApiRequest, postApiRequest } from './assets/utils';
+import { postApiRequest } from './assets/utils';
 
-let newProjectData: Project;
+const createdProject = process.env.E2ETESTS_PROJECT;
```

Comment:
Create the projectData here, then you can remove it from all the tests. Suggestion: project is a better and more simple name
```suggestion
const project: Project = JSON.parse(process.env.E2ETESTS_PROJECT!);
```